core/common/src/main/resources/alluxio-default.properties

```diff
@@ -157,6 +157,7 @@ alluxio.user.lineage.enabled=false
 alluxio.user.lineage.master.client.threads=10
 alluxio.user.network.netty.timeout.ms=3000
 alluxio.user.network.netty.worker.threads=0
+alluxio.user.ufs.operation.delegation=false
```

Comment:
High-level questions: shall we document this flag (and overall ufs delegation feature) somehwere?